uber/templates/preregistration/form.html

```diff
@@ -2,12 +2,6 @@
 {% block title %}Preregistration{% endblock %}
 {% block content %}
 
```

Comment:
Since we now move this message around dynamically based on whether it's a single or group registration, this is no longer necessary.